sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSenderAsyncClientTest.java

```diff
@@ -185,4 +194,31 @@ void sendSingleMessage() {
         final Message message = singleMessageCaptor.getValue();
         Assertions.assertEquals(Section.SectionType.Data, message.getBody().getType());
     }
+
+    @Test
+    void scheduleMessage() {
+        // Arrange
+        long sequenceNumberReturned =10;
+
+        when(managementNode.schedule(any(ServiceBusMessage.class), any(Instant.class)))
+            .thenReturn(just(sequenceNumberReturned));
+
+        // Act & Assert
+        StepVerifier.create(sender.scheduleMessage(mock(ServiceBusMessage.class), mock(Instant.class)))
```

Comment:
Its more specific you to create a mock of the message and then if the managementNode gets a schedule to return the number. any() should be used in the case when the parameter can be variable things or its not specific to the test case you are checking.